extensions/twitch.js

```diff
@@ -0,0 +1,297 @@
+// Name: Twitch
+// ID: twitch
+// Description: Blocks that allow you to stream easier on TurboWarp.
+// By: BOP <https://scratch.mit.edu/users/CEFSharp/>
+
+(function (Scratch) {
+  "use strict";
+
+  let chat = null;
+  let token = null;
+  let text = null;
+  let image = null;
+
+  class Twitch {
+    getInfo() {
+      return {
+        id: "twitch",
+        name: "Twitch",
+        color1: "#772CE8",
+        menuIconURI: "https://tekinical.github.io/twitchURI.png",
+        docsURI:
+          "https://github.com/Tekinical/tekinical.github.io/blob/main/Twitch%20Extension%20Doc.md",
+        blocks: [
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Twitch Chat",
+          },
+          {
+            opcode: "setupchat",
+            text: "setup chat [username] chat fade enabled [fade] show bots [bots] prevent clipping [prevent] width [width] height [height] position [position]",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
```

Comment:
Please make all blockIconURIs a dataURI (same as the menuIconURI)

Reply:
![image](https://github.com/TurboWarp/extensions/assets/140775902/72e5a5c3-d038-4948-abd6-cdc9f1825aac)

Reply:
That is not a dataURI